app/templates/vault.html

```diff
@@ -0,0 +1,8 @@
+<div class="vault">
```

Comment:
You have all_items.html that why this vault.html?

Reply:
All_items is for new items